server/src-lib/Hasura/SQL/DML.hs

```diff
@@ -283,7 +283,8 @@ data SQLExp
   | SELit !T.Text
   | SEUnsafe !T.Text
   | SESelect !Select
-  | SEStar
+  | SEStar !(Maybe Qual)
```

Comment:
This isn't needed any more, but it seems like a useful addition anyway. Keep or revert?

Reply:
Seems fine to keep it to me.